### DISCUSSION TITLE
Snapshot Tests not Creating Xcode Groups

### DISCUSSION THREAD

**Discussion by nikitaame** [pointfreeco/swift-snapshot-testing#767] [General]
I'm not sure if it would even be possible, but currently, whenever tests are run and recorded, directories are created but not Xcode groups. Would it be possible to create them so that `__Snapshots__` and the content within are visible directly in Xcode?

**To Reproduce**
- Record snapshots 
- Observe that the snapshot recordings aren't visible from the Xcode navigator 

**Expected behavior**
- Record snapshots 
- Observe that the snapshot recordings are visible from the Xcode navigator

**Comment by mbrandonw** [comment]
Hi @nikitaame, sorry but this is not really feasible. We do not want to make edits to the Xcode project file from the library. That can cause Xcode to crash and corrupt the project file. If you put your tests in an SPM package then this will work automatically.

Also since this is not an issue with the library but rather a feature request, I am going to move it to a discussion. Feel free to continue the discussion there.
